Versions/Common/BeStride_Mount.lua

```diff
@@ -219,28 +219,35 @@ function BeStride_Mount:DemonHunterGlide()
 end
 
 function BeStride_Mount:Druid()
-	local travelForm, flightForm = 783, 783 -- 3 in 1 travel form
+	local travelForm, flightForm = BeStride_Constants.spells.druid.travelform, BeStride_Constants.spells.druid.flightform
 
 	if GetUnitSpeed("player") ~= 0 then
-		return self:MountSpell(SpellToName(783))
+		return self:MountSpell(SpellToName(travelForm))
 	elseif BeStride:DruidFlyingMTFF() or IsFalling() or IsFlying() or GetShapeshiftForm() == 3 then
-		return self:MountSpell(SpellToName(783))
-	elseif IsFlying() then
-		return self:Flying(name)
+		return self:MountSpell(SpellToName(flightForm))
```

Comment:
Not sure if the flight form is always the best form here, and if I should not test all Shapeshift forms for WotLK.